baseapp-chats/baseapp_chats/graphql/mutations.py

```
@@ -82,6 +84,26 @@ def mutate_and_get_payload(cls, root, info, profile_id, participants, **input):
             )
 
+        if len(participants) > 2 and not is_group:
+            return ChatRoomCreate(
+                errors=[
+                    ErrorType(
+                        field="is_group",
+                        messages=[_("Is group must be true if there are more than 2 participants")],
+                    )
+                ]
+            )
+        title = input.get("title", None)
```

Ruff (0.8.2)

Use `input.get("title")` instead of `input.get("title", None)`

Replace `input.get("title", None)` with `input.get("title")`

(SIM910)